src/Silverback.Integration.Kafka/Messaging/Messages/KafkaKeyHelper.cs

```diff
@@ -1,44 +1,86 @@
-﻿// Copyright (c) 2020 Sergio Aquilini
-// This code is licensed under MIT license (see LICENSE file for details)
-
-using System;
-using System.Globalization;
-using System.Linq;
-using Silverback.Messaging.HealthChecks;
-
-namespace Silverback.Messaging.Messages
-{
-    internal static class KafkaKeyHelper
-    {
-        public static string? GetMessageKey(object? message)
-        {
-            if (message == null)
-                return null;
-
-            if (message is PingMessage)
```

Comment:
Is this necessary?
In the `KafkaMessageKeyInitializerProducerBehavior` you create a NewGuid if there is no key.
Hence, I deleted it.

Reply:
Probably not, you are right! 👍